js/ui/save-html.js

```diff
@@ -9,6 +9,22 @@ define(
         var cleanup = function (rootEl) {
             $(".removeOnSave", rootEl).remove();
             utils.removeReSpec(rootEl);
+           function attrToDataAttr(name){
+               return function(elem){
+                   var value = elem.getAttribute(name);
+                   elem.removeAttribute(name);
+                   elem.setAttribute("data-" + name, value);
+               }
+           }
+
+           var forList = rootEl.querySelectorAll("[for]");
```

Comment:
Yeah, I'm not sure what the spec says + might be tainted from my pre-qSA days when this generally broke.